packages/pathfinder/src/pathfinder.ts

```diff
@@ -0,0 +1,370 @@
+const config =  require('../../../config.json');
+const { Pool } =  require('pg');
+const pool = new Pool({
```

Comment:
This should be implemented using dependency injection, not just a static part of this file.

Reply:
not sure how to change this accordingly

Reply:
I left a top level comment with more detail.

Reply:
addressed